client/a8c-for-agencies/sections/sites/site-preview-pane/style.scss

```diff
@@ -46,6 +46,12 @@
 			height: revert;
 		}
 
+		@media (max-width: 850px) {
+			.section-nav__panel {
+				overflow: scroll;
```

Comment:
Wondering if we could be more specific here and let decide the browser engine: `overflow-x: auto;`

Reply:
Thanks for mentioning this, I looked into it more as I wasn't aware of this and it definitely appears to be the better option. Added a commit 9d15df7a7a33827fb3c0c33f10b9c28a59710267